src/radembedded.c

```diff
@@ -11,167 +11,194 @@
 
 #define AUTH_PORT_PASSWORD ":1812:testing123"
 #define ACCT_PORT_PASSWORD ":1813:testing123"
-#define SERVER_ADDR "localhost"
+#define SERVER_ADDR        "localhost"
 
-int
-main (int argc, char **argv)
+int main(int argc, char** argv)
 {
-	int		retval = 0;
-	rc_handle 	*rh = NULL;
-	uint32_t 	client_port = 0;
-	uint32_t	status_type = PW_STATUS_STOP;
-        VALUE_PAIR      *send = NULL;
+    int         retval              = 0;
+    rc_handle*  rh                  = NULL;
+    uint32_t    client_port         = 0;
+    uint32_t    status_type         = PW_STATUS_STOP;
+    VALUE_PAIR* send                = NULL;
+    VALUE_PAIR* received            = NULL;
+
+    char        username[255]       = "test";
+    char        password[255]       = "test";
+    char        callfrom[255]       = "8475551212";
+    char        callto[255]         = "8479630116";
+    char        myuuid[255]         = "981743-asdf-90834klj234";
```

Comment:
I think testing should be more comprehensive in all the options that are added. I attach a proposed patch.
[patch.txt](https://github.com/radcli/radcli/files/4420774/patch.txt)